packages/styles/src/components/form-check.scss

```
@@ -78,7 +78,7 @@
       border-radius: 50%;
 
       &:checked::after {
-        margin: spacing.$size-micro;
+        border: spacing.$size-micro solid color.$white;
```

If a default background color scheme was defined (https://github.com/swisspost/design-system/issues/2123), we could use 

```suggestion
        border: spacing.$size-micro solid rgb(var(--post-bg-rgb));
```
and have automatic dark/light background compatibility.